.github/workflows/build_deploy.yml

```diff
@@ -211,3 +211,25 @@ jobs:
           # due to a duplicate wheel being present which will ensure that the rest
           # of the wheels will be uploaded if some are uploaded manually.
           skip_existing: true
+
+  build-and-publish-init-image:
+    # We have to wait for the PyPI job since the image that we publish depends on installing
+    # the package from PyPI.
+    needs: [upload_pypi]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to Docker
+        run: docker login -u publisher -p ${{ secrets.GITHUB_TOKEN }} ghcr.io
+      - name: Docker Build
+        uses: docker/build-push-action@v3
+        with:
+          push: true
+          tags: ghcr.io/datadog/dd-trace-py/dd-lib-python-init:${{ github.ref_name }}
+          platforms: 'linux/amd64,linux/arm64/v8'
```

Comment:
Why are we publishing images for only these platforms? Do we plan to add more?

Reply:
this isn't new or changing, we only plan to support these for now